src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/BalanceCalculatorUpdateActionIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.homeoffice.digital.sas.balancecalculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.enums.AccrualType.ANNUAL_TARGET_HOURS;
+import static uk.gov.homeoffice.digital.sas.balancecalculator.testutils.CommonUtils.createTimeEntry;
+
+import java.math.BigDecimal;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.Accrual;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.accrual.enums.AccrualType;
+import uk.gov.homeoffice.digital.sas.balancecalculator.models.timecard.TimeEntry;
+import uk.gov.homeoffice.digital.sas.kafka.message.KafkaAction;
+
+@SpringBootTest
+@AutoConfigureWireMock(port = 9999)
+class BalanceCalculatorUpdateActionIntegrationTest {
+
+  private static final String TIME_ENTRY_ID = "a90a037a-a6c5-4fc0-9e72-a533c8c89c68";
+  private static final String TENANT_ID = "52a8188b-d41e-6768-19e9-09938016342f";
+  private static final String PERSON_ID = "b810a175-9b87-473c-94f3-9bcdf87f3a58";
+
+  @Autowired
+  private BalanceCalculator balanceCalculator;
+
+  @Test
+  void calculate_updateAthOneDayTimeEntry_contributionsAndCumulativeTotalsAsExpected() {
```

Comment:
is Ath a typo?

Reply:
Wanted to make Athutal Target Hours shorter as line was too long for some tests name and couldn't fit